stdlib/public/runtime/Errors.cpp

```diff
@@ -58,6 +58,10 @@
 #include <android/log.h>
 #endif
 
+#if defined(__ELF__) && (defined(__ANDROID__) || defined(__linux__)) && (defined(__arm__) || defined(__aarch64__))
```

Comment:
I think that you can leave off most of this, `__ELF__` should always be defined for Linux which covers everything:

```
#if defined(__ELF__)
```